src/common/lib/links.ts

```diff
@@ -67,3 +68,11 @@ export function createNewGithubIssueHref( link: NewGitHubIssueLink, issueTitle?:
 
 	return url.href;
 }
+
+export function createNewJiraIssueHref( link: NewJiraIssueLink ) {
+	const url = new URL( 'https://jira.tumblr.net/secure/CreateIssueDetails!init.jspa' );
+	// If the project ID or issue type is invalid, the link will just show an error saying as much.
+	url.searchParams.append( 'pid', link.project.id.toString() );
+	url.searchParams.append( 'issuetype', link.issueType.id.toString() );
+	return url.href;
```

Comment:
**Suggestion (blocking):** Hmm, I'm wondering if we want to avoid hardcoding the URL here 🤔 I believe other teams in the past other than Tumblr have used Jira, and I could see us maybe needing to broaden our support from just Tumblr here. My guess is that all new issues use the same path (`/secure/CreateIssueDetails!init.jspa`), so maybe we could add the host name of the JIRA instance (in this case, `https://jira.tumblr.net`) to the configuration type?

What do you think?

Reply:
> so maybe we could add the host name of the JIRA instance (in this case, https://jira.tumblr.net) to the configuration type? 

I see your point! I've updated the structure and added the host name for Jira. 👍🏻